docs/buildah-bud.md

```diff
@@ -406,12 +406,17 @@ that the PID namespace in which `buildah` itself is being run should be reused,
 or it can be the path to a PID namespace which is already in use by another
 process.
 
-**--platform**="OS/ARCH"
+**--platform**="OS/ARCH[/VARIANT]"
+**--platforms**="OS/ARCH[/VARIANT][,...]"
 
-Set the OS/ARCH of the image to the provided value instead of using the current
-operating system and architecture of the host (for example `linux/arm`). If
-`--platform` is set, then the values of the `--arch` and `--os` options will be
-overridden.
+Set the OS/ARCH of the built image (and its base image, if your build uses one)
+to the provided value instead of using the current operating system and
+architecture of the host (for example `linux/arm`). If `--platform` or
+`--platforms` is set, then the values of the `--arch` and `--os` options will
+be overridden.
+
+When more than one value is supplied to `--platforms`, the `--manifest` option
+should be used instead of the `--tag` option.
```

Comment:
Any way to provide a pointer to the valid values for OS and ARCH?  An added example or two would probably be beneficial too.

Reply:
Will add pointers.

Reply:
Is this a comma separate list of platforms or do I need to specify it multiple times?

Reply:
Just updated this.  Now, it's both.

Reply:
Updated it again.  `--platform` now handles everything, warning when it's given more than one value in `pull` (a single tagged reference can't be applied to multiple images, so we'll want to think a while about changing that), or `from`.

Reply:
The [docs](https://pkg.go.dev/github.com/spf13/pflag?utm_source=godoc#StringSlice) say we don't need to do splitting of comma-separated values for `StringSlice` flags ourselves, so just dropped those bits.